docs/requirements/ADP-957.md

```diff
@@ -0,0 +1,201 @@
+# Basic definitions
+
+*   **Collateral inputs**, in a transaction, are used to cover transaction fees
+    in the case that a non-native script fails. The term **collateral** refers
+    to the total ada contained in the UTxOs referenced by collateral inputs.
+    Collateral inputs can **only** contain ada: they may **not** contain
+    non-ada assets.
+
+*   The protocol parameter `collateralPercent` is used to specify the
+    percentage of the total transaction fee its collateral must (at a minimum)
+    cover. The collateral inputs must not themselves be locked by a script.
```

Comment:
> The collateral inputs must not themselves be locked by a script.

This is a requirement, and one that we are interested in too.
Coin selection must only pick VK inputs for collateral.

Reply:
> Coin selection must only pick VK inputs for collateral.

@rvl 
Thanks, I've updated the specification to make this clear.